src/dependencies.jl

```diff
@@ -106,7 +106,7 @@ end
 can_use(::Type{AptGet}) = has_apt && OS_NAME == :Linux
 package_available(p::AptGet) = can_use(AptGet) && !isempty(available_versions(p))
 function available_versions(p::AptGet)
-    vers = ASCIIString[]
+    vers = String[]
```

Comment:
I've just tagged a version locally, I'll wait for now then, especially since `Compat.ASCIIString` and `Compat.UTF8String` works even without JuliaLang/Compat.jl#198